main.cc

```diff
@@ -288,6 +288,21 @@ static void tcp_syncookies_sanity() {
     }
 }
 
+static void tcp_timestamps_sanity() {
+    try {
+        auto f = file_desc::open("/proc/sys/net/ipv4/tcp_timestamps", O_RDONLY | O_CLOEXEC);
+        char buf[128] = {};
+        f.read(buf, 128);
+        if (sstring(buf) == "0\n") {
+            startlog.error("sysctl entry net.ipv4.tcp_timestamps is set to 0.\n"
+                          "Scylla requires to set net.ipv4.tcp_timestamps=1");
+            throw std::runtime_error("sysctl entry net.ipv4.tcp_timestamps is set to 0.");
```

Comment:
@gleb-cloudius do we want to prevent startup in this case?

Reply:
changed to warning message.